### ISSUE TITLE
leader image

### ISSUE THREAD

**Issue by Dragonguard999** [Dragonguard999/stellaris-mod-great-poland#1]
[This](https://pbs.twimg.com/media/CQQ2PoGWoAAN9r4.jpg) image will be used in mod

**Comment by Bravo555** [comment]
Yes, but it's also worth to note that before it will be ready to be in game the background needs to be removed.
